src/DssSpell.t.base.sol

```diff
@@ -961,6 +965,16 @@ contract DssSpellTestBase is Config, DssTest {
                     address engine = LockstakeClipperLike(address(clip)).engine();
                     assertNotEq(engine, address(0), _concat("TestError/clip-engine-is-not-set-", ilk));
                 }
+                {
+                    // Ensure liquidation penalty is always bigger than combined keeper incentives
+                    (,,, uint256 line, uint256 dust) = vat.ilks(ilk);
+                    if (line != 0 && clip.stopped() == 0) {
+                        (, uint256 chop,,) = dog.ilks(ilk);
+                        uint256 penaltyAmount = (dust * chop / WAD) - dust;
+                        uint256 incentiveAmount = uint256(clip.tip()) + (dust * uint256(clip.chip())) / WAD;
```

Comment:
- ✅ Checked both values for precision: both are RAD
- ✅ `penaltyAmount` formula match [calculation in of the "penalty" in the dog.bark function](https://etherscan.io/address/0x135954d155898d42c90d2a57824c690e0c7bef1b#code#L228)
  - Can be simplified to `tab - art` (where `tab = chop * art * rate / WAD` and `art = dust`)

- ❌ `incentiveAmount` formula does not entirely match [calculation of the incentive inside clip.kick](https://github.com/makerdao/dss/blob/fa4f6630afb0624d04a003e920b0d71a00331d98/src/clip.sol#L261)
  - Can be simplified to `tip + tab * chip` (where `tab` is the same as above, so suppose to include rate + penalty). Relevant, since this would make `incentiveAmount` slightly bigger. Let me know if I'm making a mistake somewhere

Ideally, I think, this should be end-to-end test to avoid potential edge cases, but as we don't yet have a universal base test that can liquidate any collateral, I would put it out of scope of this PR.